### PR TITLE
add example deployment

### LINKED ISSUE #5150: Web master in ocis demo deployment
## Acceptance criteria
- Have an oCIS demo instance publicly available which is frequently (nightly?) updated to the current ocis master, but running the current web master instead of the builtin web service.
- As it's a public demo instance it needs to be reset frequently to avoid malicious usage of the storage.
- Demo users like with all other public ocis demo instances.

### PR DESCRIPTION
**Pull request by wkloucek**
## Description
Adds a continuously deployed oCIS with the latest (master) Web UI. First deployment already online at https://ocis.ocis-web.latest.owncloud.works/.
Fixes #5150 

## Open tasks
[x] Add secrets
[x] add docs to https://owncloud.dev/ocis/deployment/continuous_deployment/ -> see https://github.com/owncloud/ocis/pull/2076
[x] setup drone cron schedule

### CODE REVIEW

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUINotificationBasic</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/web/15678/

<p>


### webUINotifications-displayNotificationsOnWebUI-feature-13.png
  

### webUINotifications-displayNotificationsOnWebUI-feature-18.png
  


</p></details>

**Review by kulmann — APPROVED** [review]
Looks good to me. Changelog would be good though.

File: .drone.star [hunk 1]
```diff
@@ -546,7 +546,17 @@ def main(ctx):
     after = afterPipelines(ctx)
     dependsOn(stages, after)
 
-    return before + stages + after + checkStarlark()
+    if ctx.build.event == "cron":
+        return example_deploys(ctx)
```
Comment (phil-davis) [inline .drone.star:550]: @wkloucek this `if` statement now means that the nightly cron job does not run the full CI. It is just running `example_deploys`.

Was that intended?

Comment (wkloucek) [reply .drone.star:550]: Was not intended :-/  -> revert in https://github.com/owncloud/web/pull/5199

**Pull request merged**